packages/mollie-integration/src/mollie-payment-strategy.ts

```diff
@@ -300,18 +283,18 @@ export default class MolliePaymentStrategy implements PaymentStrategy {
         );
     }
 
-    private _mountCardVerificationfields(formOptions: HostedFormOptions): Promise<HostedForm> {
+    private mountCardVerificationfields(formOptions: HostedFormOptions): Promise<HostedForm> {
```

Comment:
why do you use different naming style for methods? `private mountCardVerificationfields` and `private _isHostedPaymentFormEnabled`
I think will be better to have one style

Reply:
fixed